src/gmail.js

```diff
@@ -2390,5 +2440,19 @@ var Gmail = function(localJQuery) {
     return false;
   }
 
+  /**
+   * Track the change of emails per page
+   * 
+   */
+  api.observe.on(
```

Comment:
Totally get ya. But what's the best way forward, here? `emails_per_page` is used by `visible_emails_pre`. If the settings are changed, this detail needs to be captured from `GLOBALS`

I'm actually doing something a bit more radical that's not in the pull-request:
Because Gmail cache's listings on the client-side, I actually reload Gmail when the settings are changed and it's for the number of emails per page. I naturally didn't include that in the PR, but I couldn't find a way to release the email-listing-caches (eg. starred-view, label-view, etc) to ensure the counts are right.